sos/plugins/postgresql.py

```diff
@@ -47,34 +48,34 @@ class PostgreSQL(Plugin):
         ('dbport', 'database server port number', '', '5432')
     ]
 
-    def pg_dump(self, pg_dump_command="pg_dump", filename="sos_pgdump.tar"):
+    def pg_dump(self, scl=None):
```

Comment:
Suggest renaming this method to `do_pg_dump()` or similar: it's annoying trying to discriminate the method name from the command name in this file.